src/engine/source/hlp2/src/parse_field.cpp

```diff
@@ -90,30 +90,34 @@ void updateDoc(json::Json& doc,
                std::string_view hdr,
                std::string_view val,
                bool is_escaped,
-               std::string_view escape)
+               std::string_view escape,
```

Comment:
The doxygen documentation is missing, and you could change the names from hdr to header? or Key?, val to value. I think it would be more readable